File: mobile/feature-headlines/src/main/java/com/rogoz208/feature_headlines/presentation/articles_list/recycler/OnArticleItemClickListener.java
package com.rogoz208.feature_headlines.presentation.articles_list.recycler;

import android.view.View;

import com.rogoz208.mobile_common.model.ArticleUI;

public interface OnArticleItemClickListener {

    void onItemClick(ArticleUI item, int position);

    void onItemLongClick(ArticleUI item, View itemView, int position);
}

File: mobile/feature-sources/src/main/java/com/rogoz208/feature_sources/articles_screen/recycler/OnArticleItemClickListener.kt
package com.rogoz208.feature_sources.articles_screen.recycler

import android.view.View
import com.rogoz208.mobile_common.model.ArticleUI

interface OnArticleItemClickListener {

    fun onArticleClick(item: ArticleUI, position: Int)

    fun onArticleLongClick(item: ArticleUI, itemView: View, position: Int)
}